services/net/notification/NotificationManager.cs

```diff
@@ -401,11 +405,14 @@ private async Task<string> GenerateNotificationBodyAsync(NotificationModel notif
     private async Task<string> GenerateNotificationAsync(NotificationModel notification, ContentModel content, bool isSubject = false)
     {
         var result = await _engine.CompileRenderStringAsync(
-            notification.Name + "_" + notification.Id,
+            notification.Name + "_" + notification.Id + (isSubject ? "_subject" : ""),
             isSubject ? (notification.Settings.GetDictionaryJsonValue<string>("subject") ?? "") : notification.Template,
             new TemplateModel
             {
-                Content = content
+                Content = content,
+                MmiaUrl = isSubject ? "" : _config.GetValue<string>("MmiaUrl"),
+                RequestTranscriptUrl = isSubject ? "" : "https://www.microsoft.ca", // TODO...
```

Comment:
Remove hardcoded values and extract values from the `Notification.Settings` values.

Reply:
It probably would be wise to simply pass the `Notification.Settings` to the `TemplateModel` and then the templates would have access to dynamic data.

Reply:
Updated, thanks